src/index.js

```diff
@@ -0,0 +1,155 @@
+
+const getLatAndLon = (city) => {
+  return axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return {
+        lat: response.data[0].lat,
+        lon: response.data[0].lon,
+      };
+    })
+    .then((response)=>{
+console.log(response);
+getWeatherKelvin(response.lat,response.lon)
+    })
+
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const getWeatherKelvin = (lat, lon) => {
+  return axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: lat,
+        lon: lon,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      return response.data.current.temp;
+    })
+    .then((response)=>{
+      console.log(response);
+      return Math.floor((response - 273.15) * 9/5 + 32)
+    })
+    .then((response)=>{
+      console.log(response);
+      state.temp=response
+      tempContainer.textContent=`${state.temp}`;
+
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+const saveCity=()=>{
+  const cityName= document.getElementById('input').value;
+  state.city=cityName
+  console.log(state.city);
```

Comment:
don't forget to take out debugging print statements like this out for final submissions

```suggestion
```